lib/puppet/ssl/validator/default_validator.rb

```diff
@@ -21,15 +20,12 @@ class Puppet::SSL::Validator::DefaultValidator #< class Puppet::SSL::Validator
   # @api private
   #
   def initialize(
-      ssl_configuration = Puppet::SSL::Configuration.new(
-                                        Puppet[:localcacert], {
-                                          :ca_auth_file  => Puppet[:ssl_client_ca_auth]
-                                        }),
-      ssl_host = Puppet.lookup(:ssl_host))
+    ca_path = Puppet[:ssl_client_ca_auth] || Puppet[:localcacert],
```

Comment:
Do we need to explicitly pass `nil` to this somewhere now, if we don't want to verify our connection?

Reply:
I tried to simplify the strung out logic for determining `ca_path` that was in the `Puppet::SSL::Configuration` initialization. The logic is essentially moved from [here](https://github.com/puppetlabs/puppet/pull/6806/files#diff-4e71133b7d184da7be067958b4a78b3eL22), and I think that it's in the correct spot now. But given your comment on the removal of `#ssl_certificates_are_present?`, I think I introduced that conditional at the wrong level, and I think it will be better to leave that method modified to use the new `@ca_path` variable.

Reply:
what's the comma at the end of the line doing?

Reply:
👍